app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -0,0 +1,85 @@
+package ar.com.wolox.android.example.ui.home.news
+
+import android.support.v7.widget.LinearLayoutManager
+import android.view.View
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.New
+import ar.com.wolox.android.example.utils.InfiniteScroll
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import kotlinx.android.synthetic.main.fragment_news.*
+import javax.inject.Inject
+
+/**
+ * Fragment for news feature;
+ */
+class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), INewsView {
+
+    @Inject lateinit var mNewsAdapter: NewsAdapter
+    @Inject lateinit var mToastFactory: ToastFactory
+
+    val TAG = NewsFragment::class.java.simpleName
+    private val VIEW_THRESHOLD = 10
```

Comment:
I think there's a big discussion by where should the constants be in Kotlin. I like to put them in the `companion object` with a `private const val`, but I think that could changes on the project.
Nevertheless, the Kotlin convention in the private properties not declared as `const` is that them should not contain underscores in the middle or in the end, so I recommend to use the `companion object` and `const`.

Reply:
So, I should put that constant inside a companion object in the fragment, shouldn't I?

Reply:
Yes. I will be something like this:
`class NewsFragment {
  [...]
  companion object {
    private const val VIEW_THRESHOLD = 10
  }
}`

As I said you before in another thing, I think where the companion object should be depends on the project. But for now, I suggest you to put it at the end of the class.